geopandas/tests/test_geom_methods.py

```diff
@@ -777,10 +778,57 @@ def test_make_valid_shapely_pre18(self):
         ):
             s.make_valid()
 
+    @pytest.mark.skipif(
+        compat.SHAPELY_GE_20,
+        reason="concave_hull is implemented for shapely >= 2.0",
+    )
+    def test_concave_hull_not_implemented_shapely_pre2(self):
+        with pytest.raises(
+            NotImplementedError,
+            match=f"shapely >= 2.0 is required, "
+            f"version {shapely.__version__} is installed",
+        ):
+            self.squares.concave_hull()
+
+    @pytest.mark.skipif(
+        not (compat.USE_PYGEOS) and not (compat.SHAPELY_GE_20),
```

Comment:
Based on the test name, I think you're intending to test the `compat.USE_PYGEOS and compat.SHAPELY_GE_20` case (but perhaps i'm misunderstanding things!). If so, shouldn't this be 
```suggestion
        not (compat.USE_PYGEOS and compat.SHAPELY_GE_20),
```
(we could also catch the warning using with `pytest.warns(UserWarning, match=...)` but it's probably not critical since the default is moving to a shapely 2.0 world.